### DISCUSSION TITLE
Error when running FDS on HPC

### DISCUSSION THREAD

**Discussion by Moabismywashpot** [firemodels/fds#12862] [Q&A]
I am trying to run the FDS program on HPC provided by my University however I am running into the following errors:

Abort(1090831) on node 0 (rank 0 in comm 0): Fatal error in PMPI_Init_thread: Other MPI error, error stack:
MPIR_Init_thread(176)
MPID_Init(1430)
MPIR_pmi_init(167)

I have had it previously running on my personal laptop without showing any errors, and I am not attempting to use MPI and only have one mesh in the script. I have also tried running a job that is not very big but still am running into the same error.

Any help would be much appreciated.

**Comment by marcosvanella** [comment]
What kind of cluster is this? Are you compiling FDS?

**Reply by Moabismywashpot** [reply]
Not sure, I think the software downloaded is the pre-compiled version of FDS

**Comment by mcgratta** [comment]
Even if you are not using MPI, you may need to start the job like this
```
mpiexec -n 1 <path to fds executable> job_name.fds
```

**Reply by Moabismywashpot** [reply]
I have run it using that command and now get an error saying 'mpiexec was unable to find the specified executable file, and therefore
did not launch the job. This error was first reported for process rank 0; it may have occurred for other processes as well.'

**Reply by mcgratta** [reply]
You need to find out from whomever installed the software where it is located on the system. If you installed it, you would have indicated where the files, including the executable, were installed.
